### PR TITLE
Bump all outdated dependencies

### PR DESCRIPTION
**Pull request by ktaletsk**
Bump minor package version to 3.1.1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jupyterlab/latex",
-  "version": "3.1.0",
+  "version": "3.1.1",
```
Comment (fcollonval) [inline package.json:3]: I would upgrade it to 3.2.0 to reflect the dependency upgrade as a patch version is expected to be a simple replacement.

**Review by fcollonval — CHANGES_REQUESTED** [review]
Thanks @ktaletsk one small comment and could you rebase this PR and update the yarn.lock please?

File: docs/changelog.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Changelog for `jupyterlab_latex`
 
+- **3.1.1**:
```
Comment (fcollonval) [inline docs/changelog.md:3]:
```suggestion
- **3.2.0**:
```

**Comment by ktaletsk** [comment]
Closing in favor of #213

**Pull request closed without merging**